clang/lib/Format/FormatTokenLexer.cpp

```diff
@@ -816,7 +816,7 @@ void FormatTokenLexer::handleTableGenMultilineString() {
   auto CloseOffset = Lex->getBuffer().find("}]", OpenOffset);
   if (CloseOffset == StringRef::npos)
     return;
-  auto Text = Lex->getBuffer().substr(OpenOffset, CloseOffset + 2);
+  auto Text = Lex->getBuffer().substr(OpenOffset, CloseOffset - OpenOffset + 2);
```

Comment:
This is actually a bug fix of https://github.com/llvm/llvm-project/pull/78032. I need this to run tests. If requested, I will split this change into single pull request.

Reply:
That would be appreciated.